roles/mev_boost/defaults/main.yml

```diff
@@ -0,0 +1,32 @@
+mev_boost_user: mev_boost
+mev_boost_datadir: /data/mev_boost
+
+mev_boost_cleanup: false # when set to "true" it will remove the container(s)
+
+################################################################################
+##
+## mev_boost container configuration
+##
+################################################################################
+mev_boost_container_name: mev_boost
+mev_boost_container_image: flashbots/mev-boost:1.5.0
+mev_boost_container_env: {}
+mev_boost_container_ports:
+  - "127.0.0.1:{{ mev_boost_container_port }}:{{ mev_boost_container_port }}"
```

Comment:
`mev_boost_container_port` is undefined. Maybe also rename it to `mev_boost_server_port` . Just to make the naming consitent to the port type it is. The naming here should be application specific (e.g. if an app has multiple ports, they should be named appropriately, e.g. "http" , "metrics".. etc. 

. As per `--help` it says:

```
  -addr string
        listen-address for mev-boost server (default "localhost:18550")
```

So I think it would be good to say that it's the "server" port.